### PR TITLE
2761-Portfolio Column Customization

### LINKED ISSUE #2761: [4][Keegan-lee] Customizable Table Component
As seen from discussion here https://github.com/bitshares/bitshares-ui/issues/2655#issuecomment-499369911

If @Keegan-lee takes this, the task is clear as already discussed.

### PR DESCRIPTION
**Pull request by Keegan-lee**
Closes #2761 

- Add Dropdown Menu with Checkboxes that allows toggling of hiding columns
- Add translations for dropdown options
- Add persistence of selected options in viewSettings
- Add styling to dropdown menu
- Add functionality to hide columns in AccountPortfolioList

Screen Shot 2019-06-19 at 4 30 45 PM

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
Seems good, now comes the hard refactor:

We need this as a HOC that wraps itself as the antd Table, and can also be used like it would be an antd Table (forwarding of props and such). Ideally, all you have to do is
```
<CustomizableTable>
    ... # any props you would be using with a antd Table
   allowToHide={true|false}  # default true
</CustomizableTable>
```
which will automatically know what columns exist and allow customization of all (no need to discriminate here, could introduce a flag for important columns, but that flag should be in table data)

File: app/components/Account/AccountPortfolioList.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ import {Tooltip, Icon as AntIcon} from "bitshares-ui-style-guide";
 import Translate from "react-translate-component";
 import AssetName from "../Utility/AssetName";
 import TranslateWithLinks from "../Utility/TranslateWithLinks";
+import _ from "lodash-es";
```
Comment (startailcoon) [inline app/components/Account/AccountPortfolioList.jsx]: Naming this to something better would probably make future debugging easier

Comment (startailcoon) [inline app/components/Account/AccountPortfolioList.jsx]: Naming this to something better would probably make future debugging easier

Comment (Keegan-lee) [reply app/components/Account/AccountPortfolioList.jsx]: I agree. I made the change.

**Comment by sschiessl-bcp** [comment]
Is this ready for re-review?

**Comment by sschiessl-bcp** [comment]
I did not see the thumbs up, sorry for delay.

**Comment by sschiessl-bcp** [comment]
I've adjusted your PR a bit

 - remove unnecessary new method in settings
 - remove unnecessary defaults (need to be dynamic, only save deviances in settings)
 - remove unncessary state for view and fix use of store listening
 - add props definition
 - remove code duplication (re-use iscolumnchecked)
 - adjust selector css and position

**Pull request merged**